js/Navigation.js

```diff
@@ -22,8 +22,8 @@ const MainNavigator = createStackNavigator(
     ShipM8: Launch,
     Login: LandingPage, // Login Page
     Clusters: Main, // Landing Page
-    // Add: AddCluster,
-    Pods: Pods
+    Pods: Pods,
+    Details: PodInfo,
```

Comment:
can we make this "PodInfo" instead of "details" to be more explicit? will also match the others